app/models/user.rb

```diff
@@ -1,5 +1,12 @@
 class User < ApplicationRecord
   has_many :microposts, dependent: :destroy
+  has_many :active_relationships, class_name: Relationship.name,
+  foreign_key: :follower_id, dependent: :destroy
```

Comment:
```suggestion
  has_many :active_relationships, class_name: Relationship.name,
    foreign_key: :follower_id, dependent: :destroy
```
sai indent, chú check lại